### ISSUE TITLE
Combined articulations which include staccato do not play staccato

### ISSUE THREAD

**Issue by rgreen5** [musescore/MuseScore#17887]
### Issue type

General playback bug

### Bug description

The playback of staccato sounds fine as does Louré. However the following combinations appear to be playing at best mezzo-staccato rather than true staccato:

* Accent staccato
* Marcato staccato

### Steps to reproduce

Play back the attached score:
[staccato.zip](https://github.com/musescore/MuseScore/files/11675721/staccato.zip)

### MuseScore Version

MuseScore version (64-bit): 4.0.2-230651545, revision: github-musescore-musescore-dbe7c6d

### Regression

Yes, this used to work in MuseScore 3.x and now is broken

### Operating system

Linux Mint 20.1

**Comment by rgreen5** [comment]
> @rgreen5 I see **Accent staccato** and **Marcato staccato** have not so short staccato, but are sure it was better in MS3?

Actually, my experience is different. In MS3 the combined staccatos play staccato OK (although the accent appears to be ignored). In MS4 the reverse is the case: the staccato is too long although the accent is preserved.

Here is the MS3 video/audio screencast:

https://github.com/musescore/MuseScore/assets/87246583/d93cba28-b220-489c-ab62-83dc4a91a3a1

**Comment by bkunda** [comment]
I think this might be an MPE issue (?).
Basically, Muse Sounds renders this more or less accurately (the "shortness" of staccato notes should basically be the same, whether as a single or combined articulation; only the velocity of the note should change with the addition of accents or marcato signs). In MS Basic and with VSTs, the "shortness" of notes with these combined articulations is actually longer than it is with staccato alone. 
Basically, we should make the "shortness" of notes with combined staccato+accent/marcato articulations the same as for notes containing only a staccato articulation.

**Comment by bkunda** [comment]
Update 11Jun24

#### MUSE SOUNDS (musesampler 0.6.3)
##### Staccato+Accent | Staccato+Marcato
- Duration: equivalent to staccato alone (Acceptable)
- Volume: equivalent to staccato alone, except Flute 1, which plays back markedly softer (Expected: they should be louder than staccato alone)

https://github.com/musescore/MuseScore/assets/86290556/0da36472-08af-4127-a83c-27fc4b7b63bb

##### Staccato+Tenuto
- Duration: _non-legato_, I.e. longer duration than staccato, but still markedly detached (Acceptable)
- Volume: roughly equivalent to staccato alone (Acceptable)

#### SOUNDFONT/VST
##### Staccato+Accent | Staccato+Marcato
- Duration: _non-legato_ articulation; I.e. longer duration than staccato, but still markedly detached (Expected: they should be at least as short as staccato alone)
- Volume: Volume increases as expected (Acceptable)

https://github.com/musescore/MuseScore/assets/86290556/13e0a3ee-e49f-4ee0-83ef-631762f92ac5

##### Staccato+Tenuto
- Duration: _non-legato_ (Acceptable)
- Volume: equivalent to staccato alone (Acceptable) 

With this in mind, I'd say that it's no longer true that "combined articulations that include staccato do not play staccato".
There's certainly some finesse work that can be done here, but these are arguably of a lower priority and should be raised separately. 

Closing for now, but happy to reopen if further issues are identified.
